using_images/sti_images/python.adoc

```diff
@@ -241,14 +246,14 @@ you can omit this parameter.
 Use the following image name structure:
 
 ****
-`openshift/[replaceable]#<platform_name>#-[replaceable]#<platform_version>#-[replaceable]#<base_builder_image>#`
+`openshift/`_<platform_name>_`-`_<platform_version>_`-`_<base_builder_image>_``
 ****
 
 Where:
 
-. [replaceable]#<platform_name># - Refers to the STI platform; for example, `python`
-. [replaceable]#<platform_version># - The version of the referenced platform, without dots; for example, `33` for Python 3.3
-. [replaceable]#<Base_builder_image># - The base OS, such as `rhel7` or `centos7`
+. `_<platform_name>_` - Refers to the STI platform; for example, `python`
+. `_<platform_version>_` - The version of the referenced platform, without dots; for example, `33` for Python 3.3
+. `_<Base_builder_image>_` - The base OS, such as `rhel7` or `centos7`
```

Comment:
@tpoitras There's an argument for this stuff being in a table, but I'm not sure if would work here. At the least, I'd suggest taking away the list, and making it into dot points. IE: getting rid of the .'s at the start of each line and putting in some *'s. A list kinda denotes a procedure. Also, under the platform_version, I'm not sure if "dots" is the right word. Maybe periods? Or is there a better, more technical name? Decimal points?